.github/workflows/lint.yml

```
@@ -83,7 +83,7 @@ jobs:
                 .replace('modules/nf-core/', '')
                 .split('/')
                 .slice(0, 2)
-                .filter(x => !x.startsWith('main.nf') && x !== 'tests' && x !== 'meta.yml' && x !== 'environment.yml' && !/^README.*\.md$/i.test(x))
+                .filter(x => !x.startsWith('main.nf') && x !== 'tests' && x !== 'meta.yml' && x !== 'environment.yml' && !/^README.*\.md$/i.test(x) && x !== 'templates')
```

```suggestion
                .filter(x => !x.startsWith('main.nf') && x !== 'tests' && x !== 'templates' && x !== 'meta.yml' && x !== 'environment.yml' && !x.endsWith('.md'))
```

sorting it a bit based on category (first direcotries, then files) and simplified the old regex to simply ignore all markdown files